.github/workflows/CI.yml

```diff
@@ -60,16 +60,16 @@ jobs:
           flake8 arkouda          
 
   arkouda_python_portability:
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-20.04
```

Comment:
I think you only need/want `20.04` for the python portability job. And I'd add a comment with something like "use older ubuntu so setup-python uses python binaries with glibc requirements chpl docker can support"